services/ui-src/src/components/fields/ChoiceListField.tsx

```diff
@@ -27,22 +31,54 @@ export const ChoiceListField = ({
   const form = useFormContext();
   form.register(name);
 
+  const [fieldValues, setFieldValues] = useState<string[] | null>(
+    props.hydrate || form.getValues(name) || null
```

Comment:
Hydrate state with db values or local form state to ensure the boxes are checked as they were/are